extension-definition-specifications/incident-ef7/Incident Extension Suite.adoc

```diff
@@ -445,6 +445,11 @@ For example, a dropper running led to a ransomware tool to be downloaded and run
 |[stixtype]#{location_url}[location]#
 |The event occurred at a specific location.
 
+|[stixtype]#<<event,event>>#
+|[stixrelationship]#performed-by#
+|[stixtype]#{identity_url}[identity]#
+|The event was performed by the identity.
+
```

Comment:
should this be "performed" like for malware and tool objects - i.e., identity performed event?

Reply:
fixed